lib/ruby_lsp/requests/hover.rb

```diff
@@ -39,10 +39,25 @@ class Hover < Listener
       def initialize(emitter, message_queue)
         super
 
+        @external_listeners = T.let(external_listeners, T::Array[RubyLsp::Listener[T.nilable(Interface::Hover)]])
```

Comment:
The type here should be
```suggestion
        @external_listeners = T.let(external_listeners, T::Array[RubyLsp::Listener[Hover]])
```
It's referring to a `Hover` the request and not `Interface::Hover`.